bsms.mediawiki

```diff
@@ -0,0 +1,176 @@
+<pre>
+  BIP: To be determined
+  Layer: Applications
+  Title: Bitcoin Secure Multisig Setup (BSMS)
+  Author: Hugo Nguyen <hugo at nunchuk.io>, Peter Gray <peter at coinkite.com>, Marko Bencun <marko at shiftcrypto.ch>, Pavol Rusnak <stick@satoshilabs.com>, Aaron Chen <aarondongchen at gmail.com>, Rodolfo Novak <rodolfo at coinkite.com>
+  Comments-Summary: No comments yet.
+  Comments-URI:
+  Status: Proposed
+  Type: Standards Track
+  Created: 2020-11-10
+  License: BSD-2-Clause
+</pre>
+
+==Introduction==
+
+===Abstract===
+
+This document proposes a mechanism to set up multisig wallets securely.
+
+===Copyright===
+
+This BIP is licensed under the 2-clause BSD license.
+
+===Motivation===
+
+The Bitcoin multisig experience has been greatly streamlined under [https://github.com/bitcoin/bips/blob/master/bip-0174.mediawiki BIP-0174
+(Partially Signed Bitcoin Transaction)]. However, what is still missing is a standardized process for setting up multisig wallets securely across different vendors.
+
+There are a number of concerns when it comes to setting up a multisig wallet:
+
+# Whether the multisig configuration, such as Signer membership, script type, derivation paths and number of signatures required, is correct and not tampered with.
+# Whether the keys or the multisig configuration are leaked during the setup.
+# Whether the Signer persists the multisig configuration in their respective storage, and under what format.
+# Whether the Signer's storage is tamper-proof.
+# Whether the Signer subsequently uses the multisig configuration to generate and verify receive and change addresses.
```

Comment:
> By saying "out of scope" what we mean is that we don't define how exactly this is achieved, it is only a recommendation. It is similar to the recommendation that HWWs should avoid XPUB reuse, without specifying the how.

"Whether the Signer subsequently uses the multisig configuration to generate and verify receive and change addresses."->"How the Signer subsequently uses the multisig configuration to generate..."

IMO, the above change makes more sense, since on Line 98 the user is told  "The Signer should subsequently use the descriptor to generate and verify receive and change addresses."  Or Line 35 should be removed, since it is the entire goal/end result of the multisig setup?

Reply:
I'll remove line 98.